server/data/cartoons/about-us.json

```diff
@@ -12,11 +12,11 @@
   "buttons": {
     "team": {
       "text": "Meet the Team",
-      "link": "https://mathsoc.uwaterloo.ca/mathsoc-cartoons-team/"
+      "link": "/mathsoc-cartoons-team/"
```

Comment:
is this link supposed to be broken? it doesn't lead anywhere rn

Reply:
I am intentionally breaking it yes 😌 

these links are to the current site. I'm redirecting them to be internal links, which will work once we have the pages implemented.